mediachain/indexer/mc_ingest.py

```diff
@@ -340,7 +340,7 @@ def tail_blockchain(via_cli = False):
 
     cur = SimpleClient()
     
-    for art in cur.read_artefacts():
+    for art in cur.get_artefacts():
```

Comment:
That makes sense -- I'm only looking at the diff so not thinking about the context too much. Let's talk about how we delineate "sketch" code like this versus "production" quality at some point.